### PR TITLE
update openapi description

### PR DESCRIPTION
**Pull request by gkwan-ibm**

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt, would you review the changes for the `summary` and `description`?

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java [hunk 1]
```diff
@@ -69,6 +70,10 @@ private Response validate(OrderRequest orderRequest) {
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "createOrder",
+               summary = "Create orders",
+               description = "This operation create orders " + 
+                   "by an OrderRequest and send to the Bar and Kitchen services.")
```
Comment (Charlotte-Holt) [inline finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java]: Update the description to
"This operation creates orders by using an OrderRequest and sends them to the Bar and Kitchen services."

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java [hunk 2]
```diff
@@ -69,6 +70,10 @@ private Response validate(OrderRequest orderRequest) {
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "createOrder",
+               summary = "Create orders",
+               description = "This operation create orders " + 
+                   "by an OrderRequest and send to the Bar and Kitchen services.")
```
Comment (Charlotte-Holt) [inline start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java]: Same thing here. Update the description to
"This operation creates orders by using an OrderRequest and sends them to the Bar and Kitchen services."

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java [hunk 3]
```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
...
     public Response serveOrder(@PathParam("orderId") String orderId) {
         return servingWindowClient.serveOrder(orderId);
     }
 
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetServingWindow",
+               summary = "Clear ready orders",
+               description = "This operation remove all orders " + 
+                   "in the ServingWindow service.")
```
Comment (Charlotte-Holt) [inline finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: Update the description to
"This operation removes all orders in the ServingWindow service."

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java [hunk 4]
```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
     @POST
     @Path("/{orderId}")
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "completeReadyOrder",
+               summary = "Complete a ready order",
+               description = "This operation completes an order " +
+                   "by the ServingWindow service.")
```
Comment (Charlotte-Holt) [inline finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: Update the description to
"This operation completes an order by using the ServingWindow service."

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java [hunk 5]
```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
     @POST
     @Path("/{orderId}")
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "completeReadyOrder",
+               summary = "Complete a ready order",
+               description = "This operation completes an order " +
+                   "by the ServingWindow service.")
```
Comment (Charlotte-Holt) [inline start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: Update the description to
"This operation completes an order by using the ServingWindow service."

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java [hunk 6]
```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
...
     public Response serveOrder(@PathParam("orderId") String orderId) {
         return servingWindowClient.serveOrder(orderId);
     }
 
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetServingWindow",
+               summary = "Clear ready orders",
+               description = "This operation remove all orders " + 
+                   "in the ServingWindow service.")
```
Comment (Charlotte-Holt) [inline start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: Update the description to
"This operation removes all orders in the ServingWindow service."

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java [hunk 7]
```diff
@@ -58,13 +62,21 @@ public Response getSingleOrder(@PathParam("orderId") String orderId){
...
     public Response getOrdersList(@PathParam("tableId") String tableId){
         return statusClient.getOrdersList(tableId);
     }
     
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetOrder",
+               summary = "Clear all orders",
+               description = "This operation remove all orders " + 
+                   "in the Status service.")
```
Comment (Charlotte-Holt) [inline finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java]: Update the description to
"This operation removes all orders in the Status service."

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java [hunk 8]
```diff
@@ -58,13 +62,21 @@ public Response getSingleOrder(@PathParam("orderId") String orderId){
...
     public Response getOrdersList(@PathParam("tableId") String tableId){
         return statusClient.getOrdersList(tableId);
     }
     
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetOrder",
+               summary = "Clear all orders",
+               description = "This operation remove all orders " + 
+                   "in the Status service.")
```
Comment (Charlotte-Holt) [inline start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java]: Update the description to
"This operation removes all orders in the Status service."

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java (hunk 3 above)
Comment (gkwan-ibm) [reply finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: updated

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java (hunk 7 above)
Comment (gkwan-ibm) [reply finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java]: updated

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java (hunk 6 above)
Comment (gkwan-ibm) [reply start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: updated

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java (hunk 8 above)
Comment (gkwan-ibm) [reply start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeStatusResource.java]: updated

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java (hunk 1 above)
Comment (gkwan-ibm) [reply finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java]: updated

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java (hunk 2 above)
Comment (gkwan-ibm) [reply start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java]: updated

File: finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java (hunk 4 above)
Comment (gkwan-ibm) [reply finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: updated

File: start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java (hunk 5 above)
Comment (gkwan-ibm) [reply start/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java]: updated

**Comment by gkwan-ibm** [comment]
@Charlotte-Holt I updated the descriptions according to your suggestion. Would you review again?

**Pull request merged**